pulsar-client-tools-customcommand-example/pom.xml

```diff
@@ -65,13 +65,6 @@
           <skip>true</skip>
         </configuration>
       </plugin>
-      <plugin>
```

Comment:
useless, it throws warning for the missing version

Reply:
please make the patch focus only on one problem (the upgrade)